requirements/pip.txt

```diff
@@ -16,12 +16,12 @@ mkdocs==0.17.3
 # Markdown 3.0 breaks with older Django Rest Framework
 Markdown<3.0
 
-django==1.9.13
+django==1.11.16
 six==1.11.0
 future==0.16.0
 
-# django-tastypie 0.13.x and 0.14.0 are not compatible with our code
-django-tastypie==0.13.0
+# django-tastypie 0.14.0 drops support for django 1.9
```

Comment:
If we are migrating to django 1.11 we may be possible to use 0.14.x here, I think.